.github/pull_request_template.md

```diff
@@ -1,15 +1,14 @@
-Ticket: DOC-<num>, <ticket-title-here>.
+Ticket: DOC-<num>
```

Comment:
```suggestion
Ticket: DOC-<num>

Site: http://docs-<hotfix/feature>-7-doc-<num>.staging.tiny.cloud/docs/tinymce/latest/
```
Might be cool to have a link to the dev site in the PR, while we're making changes.

Reply:
as in the staging branch?